manifests/fastd.pp

```diff
@@ -26,6 +28,15 @@
       ensure => file,
              notify => Service[ffnord::resources::fastd],
              content => template('ffnord/etc/fastd/fastd.conf.erb');
+    "/etc/fastd/${mesh_code}-mesh-vpn/fastd-blacklist.sh":
+      ensure => file,
+             notify => Service[ffnord::resources::fastd],
+             content => template('ffnord/etc/fastd/fastd-blacklist.sh.erb'),
+             mode => '0766';
+   "/etc/fastd/${mesh_code}-mesh-vpn/fastd-blacklist.json":
```

Comment:
this is not usable like this. as DO9XE commented already, why not remove the offending keys from the key repository? here you introduce a per gateway blacklist file, for which I do not see how we should work with that, especially when having multiple gateways it gets hard to adapt the file on all machines.